docs/config.c

```diff
@@ -3327,12 +3328,12 @@
 ** gpg --list-keys --with-colons --with-fingerprint
 ** .te
 ** .pp
-** Note: gpg's \fCfixed-list-mode\fP option should not be used.  It
+** Note: gpg's \fIfixed-list-mode\fP option should not be used.  It
```

Comment:
This is a command line option, and should be bold.

BTW, it is missing the leading `--`.